docs/source/qdrant_client.local.rst

```diff
@@ -0,0 +1,33 @@
+qdrant\_client.local package
+============================
+
+Subpackages
+-----------
+
+.. toctree::
+   :maxdepth: 4
+
+   qdrant_client.local.tests
```

Comment:
not sure whether it should be here

Reply:
This set of files is autogenerated. We don't want to delete it, but this is not rendered in the HTML